code-challenges.test.js

```diff
@@ -35,10 +35,36 @@ const num2 = 0
 const num3 = -7
 // Expected output: "-7 is not divisible by three"
 
+describe("byThree", () => {
+    it("takes in a number as an argument and returns if the number is divisible by three or not", () => {
+        expect(byThree(num1)).toEqual("15 is divisible by three")
+        expect(byThree(num2)).toEqual("0 is divisible by three")
+        expect(byThree(num3)).toEqual("-7 is not divisible by three")
+    })
+})
 
 // b) Create the function that makes the test pass.
 
-
+//Pseudo Code:
+// This program will allow the user to check if a number is divisible by three and return the result.
+// Create a function expression (byThree())
+// Create an if statement that evaluates if a given parameter is divisible by three
+// Have the function return the string interpolation `${number} is divisible by three
+/// Create an else if statement that evaluates if a given parameter is not divisible by three
+// Have the function return the string interpolation `${number} is not divisible by three
+// console log new function with challenge parameters
+
+let byThree = (number) => {
+    if(number % 3 == 0) {
+        return `${number} is divisible by three`
+    }else if(number % 3 !== 0) {
+        return `${number} is not divisible by three`
+    }
+}
```

Comment:
This looks great!  If you want to stretch yourself, possibly consider edge cases in your code and tests.